testapps/ApplicationWithParseErrors/Views/Shared/_Layout.cshtml

```diff
@@ -0,0 +1 @@
+﻿<a href="@DateTime.()>Hello world</a>
```

Comment:
It's a compilation error, but we don't show those if we have parse errors. I was trying to figure out if we could include both of these in the output, but it makes the current code a bit more complicated. I can "fix" this file.